cmake/kokkos_tribits.cmake

```diff
@@ -10,6 +10,7 @@ IF(GTest_FOUND)
   SET(KOKKOS_GTEST_LIB GTest::gtest)
 ELSE()  # fallback to internal gtest
   SET(KOKKOS_GTEST_LIB kokkos_gtest)
+  MESSAGE(STATUS "Using internal gtest copy for testing")
```

Comment:
I would drop the 'copy' ... but this is nitpicking

Reply:
Done (7f10e39799487c7ed30157a27a94764914960b8a).